### PR TITLE
Update app.js

### PR DESCRIPTION
**Pull request by Devanshk9**
[GCI-<Devanshk09>] 
Hi I have raised a PR with /info

### CODE REVIEW

**Comment by Devanshk9** [comment]
[GCI-<your-gci-id>] message

**Comment by Devanshk9** [comment]
[GCI-<Devanshk09>] message

**Comment by Devanshk9** [comment]
GCI ID - Devanshk09

File: app.js [hunk 1]
```diff
@@ -3,10 +3,20 @@ const app = express();
 
 const PORT = 3000;
 
+//defining var to get data from package.json
+var jsonfiledb = require('./package.json')
```
Comment (f4ww4z) [inline app.js]: rename this variable to something more meaningful.

Comment (Devanshk9) [reply app.js]: done

**Pull request merged**